opba-protocols/xs2a-protocol-tests/xs2a-bdd-wiremock/src/main/resources/mockedsandbox/restrecord/embedded/zero-sca/accounts/sandbox/mappings/mapping-v1-accounts-without-balance.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
This file is not needed - there is always multi-sca for the test case added by this PR

Reply:
because now some tests now pass withBalance=false wich was not found before and thus failed.

Reply:
So the existing fixtures should be modified